### ISSUE TITLE
Web Interface Updates

### ISSUE THREAD

**Issue by CannonLock** [UW-Macrostrat/web#107]
List of interface updates before next run through:

- [ ]

**Comment by davenquinn** [comment]
A lot of these are "nice to have" rather than essential but it really depends on the full end-to-end workflow. I'd rather see something that allows the task to be stumbled through with some caveats, and then tighten up aspects as we go. That's why I think the "step-by-step instructions" on how to use the interface are important.

Grouping and other things that allow real work to be accomplished quickly are the priority. Right now, we can't really make changes effectively using the UI, so it's hard to discover what the "right" ordering should be.

**Comment by davenquinn** [comment]
OK I've lightly gone through and adjusted the ranks here by priority. I'd emphasize that this list (broadly, "usability") runs in parallel to two equally large tasks:
- getting new maps in the system to try the approach
- writing documentation of the expected workflow for USGS users

**Comment by CannonLock** [comment]
Completed # 2 in 283614d3f47dea0385c8de79e3b057cfe059749e
